playbook-website/app/javascript/site_styles/main.scss

```diff
@@ -29,9 +29,9 @@
 // CHANGELOG STYLES
 @import "./changelog";
 
-@import "fonts/site-fonts";
+@import "@powerhome/power-fonts/fonts/fonts";
```

Comment:
Can we rename this file? `fonts/fonts` is strange

Reply:
renamed to "@powerhome/power-fonts/fonts/all"